host_modules/config_engine.py

```diff
@@ -0,0 +1,60 @@
+"""Config command handler"""
+
+from host_modules import host_service
+import subprocess
+import os
+
+MOD_NAME = 'config'
+DEFAULT_CONFIG = '/etc/sonic/config_db.json'
+
+class Config(host_service.HostModule):
+    """
+    DBus endpoint that executes the config command
+    """
+    @host_service.method(host_service.bus_name(MOD_NAME), in_signature='s', out_signature='is')
+    def reload(self, config_db_json):
+
+        cmd = ['/usr/local/bin/config', 'reload', '-y']
+        config_db_json = config_db_json.strip()
+        if config_db_json and len(config_db_json):
+            config_file = '/tmp/config_db.json'
```

Comment:
>config_file [](http://example.com/codeflow?start=12&length=11)

You can use the command `config load /dev/stdin -y`, no need to create a tmp file.